client/js/components/procedure/SegmentsBulkEdit/SegmentsBulkEdit.vue

```diff
@@ -486,11 +551,14 @@ export default {
   },
 
   mounted () {
-    const promises = [this.listTagTopics({ include: 'tag' }), this.listTags({ include: 'topic' })]
+    const promises = [
+      this.listTagTopics({ include: 'tag' }),
+      this.listTags({ include: 'topic' }),
+      this.fetchPlaces()
```

Comment:
I'm not sure how to handle this. In other components we also call this methid `fetchPlaces`, which is a strong argument of doing it here as well. The other methods `listTagTopics` and `listTags` emphasize the "list" thingy. Ok while writing this here i guess i tend to call it fetch because this is what the method does. it fetches places. well, in fact it fetches a list of places but that is less relevant. This leads me to think we should rename all `list...` methods that actually call `dpApi` into `fetch...`. But not today, and not without everyone having agreed on that nitpicky rule.

Reply:
Yes, I understand you and think when we call a method `fetchPlaces` it is more clear what the method does…